qos/evm/state.go

```diff
@@ -1,27 +1,39 @@
 package evm
 
 import (
-	"fmt"
 	"sync"
 
 	"github.com/pokt-network/poktroll/pkg/polylog"
 
 	"github.com/buildwithgrove/path/protocol"
 )
 
+// EndpointStoreConfig captures the modifiable settings of the EndpointStore.
+// This will enable `EndpointStore` to be used as part of QoS for other EVM-based
+// blockchains which may have different desired QoS properties.
+// e.g. different blockchains QoS instances could have different tolerance levels
+// for deviation from the current block height.
+type serviceStateConfig struct {
```

Comment:
Do we need a `TODO_REFACTOR` to ensure that we have this logic mimiced in the other qos modules as well?